### PR TITLE
[MBL-16719][Student] Offline content screen modifications

### PR DESCRIPTION
**Pull request by hermannakos**
Test plan: Check the functionality of the tri state checkbox. Check if the file selection is saved to the db correctly.

refs: MBL-16719
affects: Student
release note: none

## Checklist

- [x] Tested in dark mode
- [x] A11y checked

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

File: libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/daos/CourseSyncSettingsDaoTest.kt
Comment (kristofnemere) [inline libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/daos/CourseSyncSettingsDaoTest.kt:1]: The copyright text is missing here.

File: libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/daos/CourseSyncSettingsDaoTest.kt [hunk 1]
```diff
@@ -0,0 +1,195 @@
+package com.instructure.pandautils.room.offline.daos
```
Comment (kristofnemere) [inline libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/daos/CourseSyncSettingsDaoTest.kt:1]: The copyright text is missing here.

File: libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/daos/FileSyncSettingsDaoTest.kt [hunk 2]
```diff
@@ -0,0 +1,130 @@
+package com.instructure.pandautils.room.offline.daos
```
Comment (kristofnemere) [inline libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/daos/FileSyncSettingsDaoTest.kt:1]: The copyright text is missing here.

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/offline/offlinecontent/OfflineContentViewData.kt [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package com.instructure.pandautils.features.offline.offlinecontent
```
Comment (kristofnemere) [inline libs/pandautils/src/main/java/com/instructure/pandautils/features/offline/offlinecontent/OfflineContentViewData.kt:1]: The copyright text is missing here.

File: libs/pandautils/src/main/java/com/instructure/pandautils/room/offline/entities/FileSyncSettingsEntity.kt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+package com.instructure.pandautils.room.offline.entities
```
Comment (kristofnemere) [inline libs/pandautils/src/main/java/com/instructure/pandautils/room/offline/entities/FileSyncSettingsEntity.kt:1]: The copyright text is missing here.

File: libs/pandautils/src/main/java/com/instructure/pandautils/room/offline/model/CourseSyncSettingsWithFiles.kt [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package com.instructure.pandautils.room.offline.model
```
Comment (kristofnemere) [inline libs/pandautils/src/main/java/com/instructure/pandautils/room/offline/model/CourseSyncSettingsWithFiles.kt:1]: The copyright text is missing here.

**Review by kristofnemere — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

**Pull request merged**